### PR TITLE
test(components/forms): single file attachment visual test

### PR DESCRIPTION
**Pull request by Blackbaud-SandhyaRajasabeson**
[AB#2195440](https://blackbaud.visualstudio.com/Products/_boards/board/t/SKY%20UX%20Program/Stories/?workitem=2195440)

### CODE REVIEW

**Comment by blackbaud-sky-build-user** [comment]
[Storybook preview](https://blackbaud.github.io/skyux-pr-preview/1497/storybook/)

Component Storybooks:

- [a11y-storybook](https://blackbaud.github.io/skyux-pr-preview/1497/storybooks/a11y-storybook/)
- [action-bars-storybook](https://blackbaud.github.io/skyux-pr-preview/1497/storybooks/action-bars-storybook/)
- [ag-grid-storybook](https://blackbaud.github.io/skyux-pr-preview/1497/storybooks/ag-grid-storybook/)
- [angular-tree-component-storybook](https://blackbaud.github.io/skyux-pr-preview/1497/storybooks/angular-tree-component-storybook/)
- [avatar-storybook](https://blackbaud.github.io/skyux-pr-preview/1497/storybooks/avatar-storybook/)
- [colorpicker-storybook](https://blackbaud.github.io/skyux-pr-preview/1497/storybooks/colorpicker-storybook/)
- [core-storybook](https://blackbaud.github.io/skyux-pr-preview/1497/storybooks/core-storybook/)
- [data-manager-storybook](https://blackbaud.github.io/skyux-pr-preview/1497/storybooks/data-manager-storybook/)
- [datetime-storybook](https://blackbaud.github.io/skyux-pr-preview/1497/storybooks/datetime-storybook/)
- [errors-storybook](https://blackbaud.github.io/skyux-pr-preview/1497/storybooks/errors-storybook/)
- [flyout-storybook](https://blackbaud.github.io/skyux-pr-preview/1497/storybooks/flyout-storybook/)
- [forms-storybook](https://blackbaud.github.io/skyux-pr-preview/1497/storybooks/forms-storybook/)
- [indicators-storybook](https://blackbaud.github.io/skyux-pr-preview/1497/storybooks/indicators-storybook/)
- [inline-form-storybook](https://blackbaud.github.io/skyux-pr-preview/1497/storybooks/inline-form-storybook/)
- [layout-storybook](https://blackbaud.github.io/skyux-pr-preview/1497/storybooks/layout-storybook/)
- [lists-storybook](https://blackbaud.github.io/skyux-pr-preview/1497/storybooks/lists-storybook/)
- [lookup-storybook](https://blackbaud.github.io/skyux-pr-preview/1497/storybooks/lookup-storybook/)
- [modals-storybook](https://blackbaud.github.io/skyux-pr-preview/1497/storybooks/modals-storybook/)
- [navbar-storybook](https://blackbaud.github.io/skyux-pr-preview/1497/storybooks/navbar-storybook/)
- [pages-storybook](https://blackbaud.github.io/skyux-pr-preview/1497/storybooks/pages-storybook/)
- [phone-field-storybook](https://blackbaud.github.io/skyux-pr-preview/1497/storybooks/phone-field-storybook/)
- [popovers-storybook](https://blackbaud.github.io/skyux-pr-preview/1497/storybooks/popovers-storybook/)
- [progress-indicator-storybook](https://blackbaud.github.io/skyux-pr-preview/1497/storybooks/progress-indicator-storybook/)
- [split-view-storybook](https://blackbaud.github.io/skyux-pr-preview/1497/storybooks/split-view-storybook/)
- [storybook](https://blackbaud.github.io/skyux-pr-preview/1497/storybooks/storybook/)
- [tabs-storybook](https://blackbaud.github.io/skyux-pr-preview/1497/storybooks/tabs-storybook/)
- [text-editor-storybook](https://blackbaud.github.io/skyux-pr-preview/1497/storybooks/text-editor-storybook/)
- [theme-storybook](https://blackbaud.github.io/skyux-pr-preview/1497/storybooks/theme-storybook/)
- [tiles-storybook](https://blackbaud.github.io/skyux-pr-preview/1497/storybooks/tiles-storybook/)
- [toast-storybook](https://blackbaud.github.io/skyux-pr-preview/1497/storybooks/toast-storybook/)

Apps:

- [code-examples](https://blackbaud.github.io/skyux-pr-preview/1497/code-examples/)
- [dep-graph](https://blackbaud.github.io/skyux-pr-preview/1497/dep-graph/)
- [integration](https://blackbaud.github.io/skyux-pr-preview/1497/integration/)
- [playground](https://blackbaud.github.io/skyux-pr-preview/1497/playground/)

File: apps/e2e/forms-storybook-e2e/src/e2e/single-file-attachment.component.cy.ts [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import { E2eVariations } from '@skyux-sdk/e2e-schematics';
+
+describe('forms-storybook', () => {
```
Comment (Blackbaud-ErikaMcVey) [inline apps/e2e/forms-storybook-e2e/src/e2e/single-file-attachment.component.cy.ts]: I think we do want to add the component here--`forms-storybook - single file attachment`

File: apps/e2e/forms-storybook/src/app/single-file-attachment/single-file-attachment.component.html [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+      <sky-file-attachment-label>
+        Single file attachment
+      </sky-file-attachment-label>
+    </sky-file-attachment>
+  </div>
+</form>
+
+<form [formGroup]="fileForm">
+  <div style="margin-bottom: 20px">
+    <sky-file-attachment
```
Comment (Blackbaud-ErikaMcVey) [inline apps/e2e/forms-storybook/src/app/single-file-attachment/single-file-attachment.component.html:16]: We have some logic around required file attachments to show the red star, can you add that to the screenshot as well?

**Review by Blackbaud-ErikaMcVey — DISMISSED** [review]

File: apps/e2e/forms-storybook/src/app/single-file-attachment/single-file-attachment.component.ts [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+  styleUrls: ['./single-file-attachment.component.scss'],
+})
+export class SingleFileAttachmentComponent {
+  @Input()
+  public set uploadedFiles(value: SkyFileItem) {
+    this.reactiveFileUpdated({ file: value });
+    this.showStatesFlag = false;
+  }
+
+  public showStatesFlag = true;
```
Comment (Blackbaud-ErikaMcVey) [inline apps/e2e/forms-storybook/src/app/single-file-attachment/single-file-attachment.component.ts]: is this variable doing anything? it looks like it gets set to false but never used anywhere

**Review by Blackbaud-ErikaMcVey — APPROVED** [review]

**Pull request merged**